filebeat/module/elasticsearch/log/ingest/pipeline.json

```diff
@@ -0,0 +1,51 @@
+{
+    "description": "Pipeline for parsing the Elasticsearch standard log file.",
+    "on_failure": [
+        {
+            "set": {
+                "field": "error.message",
+                "value": "{{ _ingest.on_failure_message }}"
+            }
+        }
+    ],
+    "processors": [
+        {
+            "grok": {
+                "field": "message",
+                "pattern_definitions": {
+                    "GREEDYMULTILINE": "(.|\n)*"
+                },
+                "patterns": [
+                    "\\[%{TIMESTAMP_ISO8601:timestamp}\\]\\[%{LOGLEVEL:log.level}%{SPACE}?\\]\\[%{DATA:elasticsearch.log.component}%{SPACE}*\\]%{GREEDYMULTILINE:message}"
```

Comment:
Does this single pattern parse all messages in a fully-configured ES with X-Pack? I believe that some components (notably ML) do not log with this structure (i.e., timestamp level component node as a prefix)?

Reply:
Was not aware of that unfortunately. I hoped that at least this part here is the same for all logs. I suggest we get it in as is and fix this in a follow up PR where we also add ml test logs.